src/feature/RansacFeatureSetMatcher.cpp

```diff
@@ -36,7 +36,7 @@ bool indexedDistanceCompare(const IndexedDistance& first, const IndexedDistance&
 
 RansacFeatureSetMatcher::RansacFeatureSetMatcher(double acceptanceThreshold, double successProbability, double inlierProbability, double distanceThreshold, double rigidityThreshold, bool adaptive, bool inliersScore):
     AbstractFeatureSetMatcher(acceptanceThreshold),
-    m_successProbability(successProbability),
+    m_successProbability(std::min(std::max(successProbability, 0.0), 0.999)),
```

Comment:
with `0.0` the number of `iterations` if `m_adaptive == true` would be `0`, so we have to handle that case.